README.md

```diff
@@ -24,6 +24,8 @@ A community maintained open source project aimed at making personal portfolio fo
 
 ### Home Page:
 
+Welcome to Home Page! If you want to navigate to this page, open the Awesome-Portfolio-Webites folder in a text editor. Once this folder is open, multiple files will appear. Click on the index.html file to access the home page.
+
```

Comment:
Thanks for your contribution.
1. Please make sure to check the spelling of the content you wrote.
2. What did you mean by "open the Awesome-Portfolio-Website folder in a text editor"?
3. Please use the following markdown syntax to distinguish folder/file names from the rest of the text: use `index.html` instead of plain index.html